games-action/dxx-rebirth/metadata.xml

```diff
@@ -20,16 +20,24 @@
 		DXX-Rebirth.
 	</longdescription>
 	<use>
-		<flag name="data">Install data files</flag>
-		<flag name="descent1">Install Descent 1 resources and executable</flag>
-		<flag name="descent2">Install Descent 2 resources and executable</flag>
+		<flag name="d1x">Install Descent 1 resources and executable</flag>
+		<flag name="d2x">Install Descent 2 resources and executable</flag>
+		<flag name="descent1">Install Descent 1 resources and executable (used prior to version 0.61)</flag>
+		<flag name="descent2">Install Descent 2 resources and executable (used prior to version 0.61)</flag>
+		<flag name="data">Install the playable demo data (i.e. shareware levels)</flag>
+		<flag name="debug">Enable various debugging code paths and sanity checks, not just output. Some of these checks date back to the retail build, and choke on some custom levels because the game was never coded to sufficiently distrust inputs</flag>
+		<flag name="editor">Enable level editor support</flag>
 		<flag name="l10n_de">Provides German translations for the game's briefings, robot descriptions, and outro of the original campaign</flag>
+		<flag name="midi">Enforce that SDL-mixer is used and has midi support</flag>
 		<flag name="music">Enable music support</flag>
 		<flag name="opl3-musicpack">Include alternate version of the game's music, recorded with the Yamaha YMF262 OPL3 soundchip</flag>
 		<flag name="sc55-musicpack">Include alternate version of the game's music, recorded with the Roland Sound Canvas SC-55 MIDI module</flag>
-		<flag name="textures">Include high resolution (640x480) backgrounds, briefing images and fonts (Descent 1)</flag>
+		<flag name="sdl2">Use libSDL2, SDL2-mixer instead of libSDL-1* and SDL-mixer-1*</flag>
+		<flag name="tracker">Enable support for the PHP-based multiplayer game tracker</flag>
+		<flag name="textures">Include high resolution (640x480) backgrounds, briefing images and fonts (for Descent 1)</flag>
+		<flag name="valgrind">Include a Valgrind header in build</flag>
```

Comment:
That is the immediate consequence, but that is not why users would care.  USE=valgrind requires the Valgrind header because it enables use of Valgrind macros to add [Memcheck client requests](https://www.valgrind.org/docs/manual/mc-manual.html#mc-manual.clientreqs) in various places.  These requests mostly deal with marking memory as undefined when Valgrind's normal lifetime rules could not discover that automatically.

Reply:
Are memcheck client requests the only reason anyone would add a Valgrind header, or is this just one feature of Valgrind amongst many others that your code is leveraging? In other words, does the USE flag need to give a specific explanation of how the code leverages Valgrind?

Reply:
Updated USE flag description